src/editor/ui/hierarchy.rs

```diff
@@ -118,8 +118,8 @@ fn draw_entity(
     };
 
     let entity_name = name.map_or_else(
-        || format!("Entity {:?}", entity),
-        |name| format!("Entity {:?}: {:?}", entity, name.as_str()),
+        || format!("Entity ({:?})", entity),
+        |name| format!("{} ({:?})", name.as_str(), entity),
```

Comment:
Not sure why this is here, but should be ignored once merged

Reply:
I collapsed the pull request into a single commit for greater readability. It might be better to do a regular merge. My pull request contains 33 commits when it should contain 3 because of this 😅

Reply:
This one in particular should just disappear as they have the same history, but yeah, I usually like to squash merge and add the specific features to the commit. There is always a lot of FMT and CI commits 😂